euphoria-core/src/main/java/cz/seznam/euphoria/core/client/operator/Join.java

```diff
@@ -442,7 +442,7 @@ public boolean isOuter() {
         OUT, JoinState, W> reduce;
 
     name = getName() + "::ReduceStateByKey";
-    reduce = new ReduceStateByKey<>(
+    reduce = new ReduceStateByKey(
```

Comment:
It's just an eclipse problem. I included these chnages (~5-10) inside this PR, should I open special one for this?